### PR TITLE
Copy tweaks

### PR DESCRIPTION
**Pull request by krissy**
### Description

Some content tweaks for sections: Homepage, Non Profits, Events, FAQ.

**_Progress: 100%**_

---
### Tasks
- [x] Update headings on homepage under **Who We Are** so it reads better as descriptions of who our org is. Was: `Women In Tech / Open Source / Our Mission`. Now: `Women In Tech / Community-Driven / Matchmakers`. 
- [x] Update copy across the content pages, fixing typos and improving wording so it's clear and more specific to the revised WHFNP mission
### Screengrabs

screen shot 2016-02-09 at 00 02 47

### CODE REVIEW

File: src/faq.html [hunk 1]
```diff
@@ -80,26 +80,26 @@ <h2 class="section-heading">Volunteers: Frequently Asked Question</h2>
...
         <h3>5. I’m new to coding/technology or returning after a long time - can I still join this group?</h3>
         <p>Yes, of course! One of our key goals is to aid the development of coding and technical skills within WHFNP. We have built a community that prides itself on supporting each other through sharing resources, providing workshops and simply helping each other out while we collaborate. Choose your project, tell your team that you are a “newbie” or “rusty” (a lot of us are!) and they’ll be more than happy to help. All you need to join is a passion for technology, open source and making great projects happen.</p>
       </div>
     </section>
     <section>
       <div class="faq">
-        <h2 class="section-heading">Non Profits: Frequently Asked Question</h2>
+        <h2 class="section-heading">Frequently Asked Questions: Non Profits</h2>
```
Comment (tanyapowell) [inline src/faq.html]: "_Non Profits_" should be "_Non-Profits_" to keep spelling consistent :)

**Comment by lili2311** [comment]
Can we add one more sentence to index.html Who We Are > Women in Tech column? Just to even out the amount of text a little more. Perhaps we can mention we run regular events and workshops? cc @krissy 

Other than that :+1:  on changes so far, are the rest of the changes to be added in this PR or a new one?

**Comment by krissy** [comment]
Ta @lili2311, updated (see screengrab) and moved pending tasks to their own [issue](https://github.com/womenhackfornonprofits/whfnp/issues/8).

**Comment by khandelwalrinki** [comment]
Looks good to me. I agree we have to have a guideline in place for allowing non profits access to Slack.

**Comment by lili2311** [comment]
:+1: lets make a Jira ticket for Slack copy and get a decision on what is going to be said there.

**Comment by krissy** [comment]
:cactus: Rebased on master and force pushed :cactus: 

If you've checked this out locally, run: `git fetch && git reset --hard origin/fix/copy-updates`

**Pull request merged**